app/views/customers/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h2><%= @customer.name %></h2>
+<br>
+<% @items.each do |item| %>
```

Comment:
Here is where we can just call `@customer.items` and get rid of that extra line in your controller.